R/validate.R

```diff
@@ -3,12 +3,14 @@
 #' Checks that user inputs are within defined ranges.
 #'
 #' @inheritParams run_lrisk
+#' @param carbon_price_model Character vector, indicating which NGFS model is used in regards to
+#'   carbon prices. Default is no carbon tax.
 #' @param risk_type String that is either lrisk or trisk.
 #' @return NULL
 validate_input_values <- function(baseline_scenario, shock_scenario, scenario_geography,
```

Comment:
the conditions in this function do not look entirely correct to me,
1. carbon_price_model is a character variable and needs to be added explicitly in the variables on which validate_values_in_values is called (to make sure this is done correctly I would suggest adding a unit test where you use an incorrect carbon tax name)
2. carbon_price_model is not in vector_numeric_args from the start so should not have to be removed from there

Reply:
done however problems with the test

Reply:
so what i dont understand rn is that the validate_values_in_values test fails i have no idea what i am doing wrong - is there a mistake in the function ?